src/uct/sm/mm/posix/mm_posix.c

```diff
@@ -674,7 +682,7 @@ static uct_mm_md_mapper_ops_t uct_posix_md_ops = {
         .mem_free               = uct_posix_mem_free,
         .mem_advise             = ucs_empty_function_return_unsupported,
         .mem_reg                = ucs_empty_function_return_unsupported,
-        .mem_dereg              = ucs_empty_function_return_unsupported,
+        .mem_dereg              = uct_posix_mem_dereg,
```

Comment:
why changed?

Reply:
because it should check for INVALIDATE flag - in case if user passed INVALIDATE flag and function return "success" user may expect that callback will be called

Reply:
since registration is not supported, dereg is also not supported - no matter which flag was passed

Reply:
I see, reverted